src/Components/Navigation/NavBar.jsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom'
+import style from '../Navigation/NavBar.module.css'
+
+function NavBar() {
+    return (
+        <div>
+            <nav className={style.navbar}>
+                <div className={style.logo}>FE NC NEWS</div>
+
+                <ul className={style.nav_links}>
+                    <div className={style.menu}>
+                        <li>
+                            <Link to="/">Home</Link>
+                        </li>
```

Comment:
symatically a `<ul>`s first child should be an `<li>` (not a `<div>`!)

if you need an extra container consider switching the tags around - otherwise you could remove the `<div>` and apply styles only to the `<ul>`